app/frontend/src/components/FileInput.vue

```diff
@@ -210,7 +212,7 @@ export default {
       templateTab: null,
       contextTab: null,
       form: {
-        contexts: null,
+        contexts: '[{}]',
```

Comment:
Does this need to be started up as a valid empty object?

Reply:
helps